server/schema/TypeDefs.ts

```diff
@@ -27,7 +26,7 @@ const typeDefs = gql`
         depth: Int!
         complexity: Int!
         tokens: Int!
-        blocked: Boolean!
+        success: Boolean!
         timestamp: Int!
```

Comment:
The `logged_on` field in the schema did not make it here.

Reply:
will address this today